corehq/toggles.py

```diff
@@ -6,9 +6,10 @@
 
 
 class StaticToggle(object):
-    def __init__(self, slug, label, namespaces=None):
+    def __init__(self, slug, label, namespaces=None, documentation_link=None):
```

Comment:
minor, but feature preview uses `help_link`. may be better to keep it consistent to reduce any dumb bugs when migrating from toggle to feature preview